### DISCUSSION TITLE
Silent Mode in G-Helper is not "silent"

### DISCUSSION THREAD

**Discussion by j4yr0s** [seerge/g-helper#2677] [General]
I really want to get rid of Armoury Crate, but it seems I'm tied to it.

I have an old ASUS TUF FX505DT, from 2019, with an AMD Ryzen 7 3750H CPU. When I select **“Silent” mode inside Armoury Crate**, the CPU stays inside these values:

- Frequency: 1380 MHz - 2800 MHz.
- Voltage: 690mV - 900mV
- Temperature: 41°C - 60°C
- CPU Fan: 2700RPM more or less.

If I uninstall Armoury Crate and I start using the default Windows 11 power modes, the CPU speeds and temps are all over the place. Even in "Power Save" mode, the CPU can easily be at over 3000 MHz most of the time. The temps are always above 55°C and the CPU Fan speed is constantly over 3500RPM, with the Voltage constantly reaching 1000mV, so **there is no way to have a "Silent" laptop just with the Windows Power Modes**, without using Armoury Crate.

The annoying thing is that G-helper has the same behavior as only using the Windows 11 Power Modes. Silent does not “force” the CPU to stay at around 700mV, the temps under 60C and the fans under 3500RPM. Also, there's no way to disable Boost, neither with the Windows regedit nor with G-helper. The base frequency of this CPU is supposed to be 2.3GHz with a boost frequency of 4.0GHz, but "deactivating" boost from Windows or G-helper does nothing, and forcing deactivating from regedit automatically limits the CPU to 1.6Ghz. It's a mess of a laptop, thank you, ASUS.

So far, I'm forced to keep using Armoury Crate to have a "Silent" laptop.

I imagine all this is tied up to the BIOS, so G-helper developers can't do much, but it doesn't hurt to ask, right?

Now, things got a little bit worst in recent months. Apparently, some update of Windows 11 23H1 or 23H2 did some change in how the Power Modes work and the newer versions of Armoury Crate stopped controlling the performance modes. "Silent" wasn't limiting the CPU voltage to around 700mV any longer.

I reported this in the ASUS Armoury Crate forum over and over again, until one of the moderators understood my issue and send my this file. They told me to execute the "ConfigInstaller.exe" inside that zip file with Administrator rights on Windows, and that fixed the issue with Armoury Crate. “Silent” was silent again. I don't know what does those files does, but I'm sharing here in the rare case they say something useful for a developer.

[ACConfig_FX505DT_1.0.0.zip](https://github.com/user-attachments/files/15527451/ACConfig_FX505DT_1.0.0.zip)

I wish there was an app like G-Helper that could allow me to set my own parameters to each performance mode, something like:

- Silent: voltage 690mV to 890mV, frequency 1399MHz to 2800MHz, fan 2000RPM to 3500RPM
- Performance: voltage 690mV to 1100mV, frequency 1399MHz to 3600MHz, fan 2000RPM to 4500RPM
- Turbo: voltage 690mV to 1350mV, frequency 1399MHz to 4000MHz, fan 2000RPM to 5500RPM

But I imagine the BIOS is the one not allowing to have a manual configuration.

I hope I was clear with my issue, since English is not my first language.

Thanks to whoever is reading.

**Comment by seerge** [comment]
@j4yr0s hello,

**Modes are hardcoded in BIOS**, not in the apps. AC or G-Helper just switch between them with a single simple command.

You even run this command by yourself in poweshell, where ``Control_status`` can be 0 (balanced), 1 (turbo), 2 (silent).
```
Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -ClassName AsusAtkWmi_WMNB) -MethodName DEVS -Arguments @{Device_ID=0x00120075; Control_status=0}
```

All this frequencies, voltages, etc are not defined or controlled in the app anyhow.

If something is wrong with your power plan in windows, i can only advice to [reset it to defaults](https://github.com/seerge/g-helper/wiki/Troubleshooting#resetting-windows-power-plan-to-defaults)

Thanks.

**Reply by j4yr0s** [reply]
Thanks for the answer.

I think that I can summarize my current issue.
- When I'm using Armoury Create, the modes work, and "Silent" keeps the CPU most of the time at 700mV, 1390Mhz, 2400RPM.
- When I'm using only G-Helper (not Armoury Crate previously installed), the different modes have no effect. So, If I select "Silent" inside G-helper, the CPU stays constantly at around 1000mV, 3000Mhz and the fans go 4000RPM. It's like the "Silent" mode its not being applied.
- Exactly the same as with G-hepler when using only the Windows Power Modes and select "Power Saver" (no armoury crate or g-helper previously installed).


Running that command in Power Shell with administrator rights seem like it doesn't have the intended effect, neither. It seems to change the fans speed, but only that. The voltage its not limited by the command. Again, it's like the "Silent" mode its not being applied.

So far, it seems like this laptop only limits its voltage to 700mV in "Silent" mode when using Armoury Crate. It's not limiting the voltage to 700mV when using G-helper or that PowerShell command.

I can attach an AC log file if its worth something.

[Log_20240603.zip](https://github.com/user-attachments/files/15541021/Log_20240603.zip)

**Reply by seerge** [reply]
@j4yr0s AC also sets own proprietary power plans. You can save them separately and just use. It could be that on your (old) model those power plan have some tricky setting limiting CPU clocks or so in windows.

You can read on how to assign any custom power plan to modes in G-Helper in readme
https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode

**Reply by j4yr0s** [reply]
> @j4yr0s AC also sets own proprietary power plans. You can save them separately and just use. 

Any idea of how I could save them separately? I don't have idea how to "extract" them from AC.

Thanks

**Reply by seerge** [reply]
You can export them with powershell commands https://www.elevenforum.com/t/export-and-import-power-plan-in-windows-11.6925/
